src/metpy/calc/tools.py

```diff
@@ -1159,7 +1175,7 @@ def second_derivative(f, **kwargs):
 
 
 @exporter.export
-def gradient(f, **kwargs):
+def gradient(f, coordinates=None, deltas=None, axes=None):
```

Comment:
I just copied the existing ordering in the docstring without giving it a second thought! It would make much more sense for these to be consistent with first/second derivative.